### ISSUE TITLE
Selenium IDE & Chrome crashes on click on the FileName Edit field

### ISSUE THREAD

**Issue by jobs4mvr** [SeleniumHQ/selenium-ide#1131]
## 🐛 Bug Report

Selenium IDE & Chrome crashes on click on the Filename Edit field.

## To Reproduce 

(1) Open Google Chrome.
(2) Open Selenium IDE Extension from Chrome.
(3) Open a .SIDE file.
(4) Click on the Filename 'Edit' icon on the top Left side of the Selenium IDE App.
(5) Click inside the Filename 'Edit' field in Selenium IDE.

## Expected behavior

The User should be able to change the .SIDE Filename and save the .SIDE file to the computer.

## Project file reproducing this issue (highly encouraged)

IP restrictions - unable to include file.

## Environment

OS: Windows 10 Pro
Selenium IDE Version:  Selenium IDE Version 3.17.0
Selenium SIDE Runner Version:  Not Applicable
Node version: Not Applicable
Browser: Google Chrome
Browser Version:  Version 85.0.4183.83 (Official Build) (64-bit)

Am attaching a GIF file (converted from an AVI screen recording using the AVI to GIF converter at https://www.onlineconverter.com):

![SeleniumIDE+GoogleChromeCrashes](https://user-images.githubusercontent.com/2760106/91604189-397ca100-e93c-11ea-88b1-f07889907d0c.gif)

**Comment by jobs4mvr** [comment]
Any update on this issue? It's recurring on a consistent basis...

**Comment by jobs4mvr** [comment]
Any update on this issue yet? The issue is continuing to occur, and does so very consistently :(

**Comment by jobs4mvr** [comment]
Any updates on this issue?

**Comment by hadgithub777** [comment]
Any updates on the fix for this issue?

**Comment by aliwasif177** [comment]
Any updates on the fix for this issue?

**Comment by toddtarsi** [comment]
Oh wowie, I didn't even know about this. I just put out a version of v3 last week, but this was likely still in.

Is this crash still occurring on Windows machines for you guys? It's a little spooky to me that Chrome goes down as well. Something there ain't good.

**Comment by toddtarsi** [comment]
@aliwasif177 - I am able to edit the project without issue on Mac.

**Comment by toddtarsi** [comment]
@aliwasif177 - Are you still encountering this crash on the latest version of Chrome / Windows / Selenium IDE? I would expect not, but maybe it is still occurring.

**Comment by aliwasif177** [comment]
yes I am still encountring this issue in mac , windows and in linux as well. It work fine on normal or mediocre webs. But in extensive dom it become unresponsive and shows a lot of voilation and delay warnings in console as well as slow down the web too much .
